account_region/config-aggregator.tf

```diff
@@ -0,0 +1,44 @@
+#
+# IAM Role
```

Comment:
Does this mean we get an aggregator for every region we run this module in?

Reply:
That's a good point, to be honest I had it originally in the same block because I didn't ave the account folder and I had and it was controlled by variables but yeah you're probably only gonna want it in eu west 1. I think I'll move it, yes